### PR TITLE
Update docs to be more user-friendly

### PR DESCRIPTION
**Pull request by tbouron**
A live preview is available there: https://tbouron.github.io/clocker/

### CODE REVIEW

File: tutorials/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+</ul>
+
+<div class="tab-content">
+<div id="amp" class="tab-pane fade in active">
+
+1. [Download AMP](http://www.cloudsoft.io/amp-container-service-early-access)
+2. Install AMP. For more detailed instructions, see the [docs](http://docs.cloudsoft.io/tutorials/tutorial-get-amp-running.html#install-cloudsoft-amp).
+3. Open the AMP UI in your favourite web browser (as per the detailed instructions within the link above) and [setup a location](http://docs.cloudsoft.io/tutorials/tutorial-get-amp-running.html#add-a-location) for where you want to deploy to (e.g. your preferred cloud, or a list of IPs for pre-existing machines). We came up with location's templates to add to your catalog, that you can use out of the box for [AWS](tutorials/locations/aws-example-location.bom), [SoftLayer]((tutorials/locations/sl-example-location.bom)), [Azure](tutorials/locations/azure-example-location.bom), [GCE](tutorials/locations/gce-example-location.bom) and [Blue Box](tutorials/locations/bb-example-location.bom). For more information, head to the [tutorial section](tutorials/swarm-cluster.html#setup-a-cloud-location).
```
Comment (m4rkmckenna) [inline tutorials/getting-started.md]: The location links are broken in https://tbouron.github.io/clocker/tutorials/getting-started.html 
do they need to start with `./`

**softlayer** remove extra `()`

File: tutorials/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+1. Install Apache Brooklyn. For more detailed instructions, see the [docs](http://brooklyn.apache.org/v/latest/start/running.html#install-apache-brooklyn).
+2. Download the required Clocker files, and add them to the Brooklyn `./lib/dropins/` folder (though these are jar files, the Clocker jars just package resources such as YAML files rather than Java code):
+ * [brooklyn-etcd](https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=io.brooklyn.etcd&a=brooklyn-etcd&v=2.3.0-SNAPSHOT)
+ * [common](https://oss.sonatype.org/service/local/artifact/maven/redirect?r=releases&g=io.brooklyn.clocker&a=common&v=2.0.0)
+ * [swarm](https://oss.sonatype.org/service/local/artifact/maven/redirect?r=releases&g=io.brooklyn.clocker&a=swarm&v=2.0.0)
+ * [kubernetes](https://oss.sonatype.org/service/local/artifact/maven/redirect?r=releases&g=io.brooklyn.clocker&a=kubernetes&v=2.0.0)
+3. [Launch Brooklyn](https://brooklyn.apache.org/v/latest/start/running.html) and add the Clocker blueprints to the catalog by using the [`clocker.bom` file](clocker.bom) that lists the catalog items:
```
Comment (m4rkmckenna) [inline tutorials/getting-started.md]: broken link to clocker.bom

File: tutorials/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+ * Alternatively, if you'd like to use the web-console, choose the "Composer" tab, click the "Catalog" button, paste the contents of the `clocker.bom` file into the online editor, and click "Deploy". For more details, see the [docs](http://brooklyn.apache.org/v/latest/ops/catalog/).
+4. [Setup a location](http://brooklyn.apache.org/v/latest/ops/locations/) for where you want to deploy to. This can be done by defining your location configuration in a .bom file and deploying using the `br` CLI. Alternatively, the web-console can be used: click on the Catalog tag, then the "+" button", choose "Location", and follow the instructions in the wizard. We came up with location's templates to add to your catalog, that you can use out of the box for [AWS](tutorials/locations/aws-example-location.bom), [SoftLayer]((tutorials/locations/sl-example-location.bom)), [Azure](tutorials/locations/azure-example-location.bom), [GCE](tutorials/locations/gce-example-location.bom) and [Blue Box](tutorials/locations/bb-example-location.bom). For more information, head to the [tutorial section](tutorials/swarm-cluster.html#setup-a-cloud-location).
```
Comment (m4rkmckenna) [inline tutorials/getting-started.md]: As above location links broken

**Comment by tbouron** [comment]
@m4rkmckenna Comments addressed

**Pull request merged**